test.sh

```diff
@@ -48,20 +61,14 @@ for dir in ${VERSIONS}; do
   fi
 
   if [ -n "${TEST_OPENSHIFT_4}" ]; then
+    export IMAGE_NAME="${NAMESPACE}${BASE_IMAGE_NAME}-${dir//./}-${OS}"
+
     if [[ -x test/run-openshift-remote-cluster ]]; then
         VERSION=$dir test/run-openshift-remote-cluster
     else
       echo "-> Tests for OpenShift 4 are not present. Add run-openshift-remote-cluster script, skipping"
     fi
   fi
 
-  if [ -n "${TEST_OPENSHIFT_MODE}" ]; then
-    if [[ -x test/run-openshift ]]; then
-      VERSION=$dir test/run-openshift
-    else
-      echo "-> OpenShift tests are not present, skipping"
-    fi
-  fi
-
```

Comment:
This part runs Openshift 3 locally, I do not think we want to remove these right?

Reply:
Stupid typo :(